app/src/main/java/com/sdp13epfl2021/projmag/model/Tag.kt

```diff
@@ -0,0 +1,23 @@
+package com.sdp13epfl2021.projmag.model
+
+import java.util.*
+
+/**
+ * Class for tags, they are stored in lowercase and without spaces
+ *
+ * @property name : name of the tag
+ */
+data class Tag(var name : String) {
+    private var accepted : Boolean = false
+    init{
+        name = name.toLowerCase(Locale.ROOT).replace(" ","")
```

Comment:
I think you should also remove tabs and new lines as if I enter : "hello    world"(with a tab between hello and world) the replace function wont remove the tab, and we will have tags with tabs and new lines in the tag set.

Reply:
Fixed, did a regex to hunt for all the kinds of ways we can add spaces instead of just spaces.

Reply:
fixed with a regex that hunts for all the ways you can actually add spaces

Reply:
Nice! I will approve the PR